.github/workflows/publish_docker_images_cron.yml

```diff
@@ -20,15 +20,20 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Clone tools branch
-        run: git clone --depth=1 --branch update_properties_fixes https://github.com/citusdata/tools.git tools
```

Comment:
I suggest you create a tag on tools repo, and use that instead.

Using develop branch is prone to errors in the future. Also tags are cheap, let's bump a new version on tools.